config/config.go

```diff
@@ -81,3 +82,16 @@ func GetIAMArn(awsAccountID, awsResourceType, awsResourceID string) (arn string)
 	arn = strings.Join([]string{"arn:", awsAPI, ":iam::", awsAccountID, ":", awsResourceType, "/", awsResourceID}, "")
 	return
 }
+
+func SetHiveShardName(configMap *corev1.ConfigMap) error {
```

Comment:
I don't see where we're calling this.  I think we need to explicitly call this in manager.go first.